study.md

```diff
@@ -46,21 +46,28 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+I'm not sure that I fully understood this, but I think that "view" is what is displayed to the client, while "view-state" is an intermidiate stage that gets information from the view and redisplay the content.
 ```
```

Comment:
It sounds like you understand what's going on here. View state is referring to the data from the model that is being displayed to the user in the current view. Front end routers manage the view state based on interaction from the user.